### ISSUE TITLE
Global namespace polluting

### ISSUE THREAD

**Issue by GoogleCodeExporter** [sukanyapoolakanda/jquery-datatables-column-filter#63]
```
What steps will reproduce the problem?
1. Using oTable var name in global namespace
2. Use columnFilter function

What is the expected output? What do you see instead?
Don't pollute vars in global naespace

What version of the product are you using? On what operating system?
1.4.7

Please provide any additional information below.
Activate column filtering on the object the function was called on (as more 
tables with different configurations could be used), and don't pollute global 
namespace

Additional polluters: iRow, labelBtn, aoFilterCells
```

Original issue reported on code.google.com by `pi...@pilou.hu` on 26 Mar 2012 at 2:04

**Comment by GoogleCodeExporter** [comment]
Original comment by `joc...@gmail.com` on 4 Sep 2012 at 7:46
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Hi,

I have fixed these issues in version 1.5.0.

Thanks,
Jovan
```

Original comment by `joc...@gmail.com` on 4 Sep 2012 at 8:01
- Changed state: **Fixed**
